src/index.js

```diff
@@ -1,7 +1,129 @@
-import UI from '../modules/UI.js';
 import './style.css';
 
-const ui = new UI();
+const taskInput = document.querySelector('.add-input');
+const taskBox = document.querySelector('.task-box');
 
-// trigger the renderTodos method to display the li
-ui.renderTodos();
+let editId;
+let isEditedTodo = false;
+
+// get data from the localstorage
+const todos = JSON.parse(localStorage.getItem('todo-list')) || [];
+
+function showTodos() {
+  taskBox.innerText = '';
+  todos.forEach((todo, index) => {
+    // apply the completed status iff the task/todo is completed
+    const isCompleted = todo.completed ? 'completed' : '';
+    const checked = isCompleted === 'completed' ? 'checked' : '';
```

Comment:
- [ ] So far, excellent work 👍🏻  however  It's one of the requirements to have a new JavaScript file for the new functionality, kindly create a new JavaScript file that will handle the CRUD (create, update, delete) methods